components/pricing-select.html

```diff
@@ -0,0 +1,101 @@
+<link href="https://unpkg.com/tailwindcss@^1.0/dist/tailwind.min.css" rel="stylesheet">
+<section class="relative min-h-screen py-16 bg-gray-200 min-w-screen animation-fade animation-delay">
+    <div class="container w-full px-10 mx-auto xl:w-1/2 sm:px-5">
```

Comment:
I would remove the `xl:w-1/2`, component looks fine without it.
```suggestion
    <div class="container w-full px-10 mx-auto sm:px-5">
```